Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -0,0 +1,77 @@
+- type: entity
+  id: 50CartridgeArmorPiercing
+  name: .50 depleted uranium cartridge
+  parent: 50CartridgeBase
+  description: An armor-piercing .50 L85 cartridge, tipped with depleted uranium.
+  components:
+  - type: CartridgeAmmo
+    proto: 50BulletArmorPiercing
+  - type: Sprite
+    sprite: _Mono/Objects/SpaceArtillery/50_armorpiercing_machinegun_casing.rsi
+    layers:
+    - state: base
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 5

(indentation)